test/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDaoTest.java

```diff
@@ -60,15 +66,35 @@ public void canConsentToStudy() {
         assertTrue(userConsentDao.hasConsented(HEALTH_CODE, new StudyIdentifierImpl(consent.getStudyKey())));
 
         // Withdraw
-        userConsentDao.withdrawConsent(HEALTH_CODE, STUDY_IDENTIFIER);
+        userConsentDao.withdrawConsent(HEALTH_CODE, STUDY_IDENTIFIER, DateTime.now().getMillis());
         assertFalse(userConsentDao.hasConsented(HEALTH_CODE, new StudyIdentifierImpl(consent.getStudyKey())));
+        
+        // Cannot be retrieved (it's "not there")
+        UserConsent found = userConsentDao.getActiveUserConsent(HEALTH_CODE, STUDY_IDENTIFIER);
+        assertNull(found);
```

Comment:
This actually tests withdrawn consent, which is the same as the next test. What we actually need is a test for the consent record not being there at all (user never consented).